src/app/applications/applications.controller.app.js

```diff
@@ -2,11 +2,12 @@
   'use strict';
 
   angular
-  .module('roboconf.applications')
+  .module('roboconf.applications', ['ngCropper'])
   .controller('SingleApplicationController', singleApplicationController);
 
-  singleApplicationController.$inject = ['rClient', '$scope', '$routeParams', '$window', 'rUtils'];
-  function singleApplicationController(rClient, $scope, $routeParams, $window, rUtils) {
+  singleApplicationController.$inject = ['rClient', '$scope', '$routeParams', '$window', '$timeout',
```

Comment:
Do you mean that directly using `$scope.$broadcast($scope.showEvent);` does not work?
Because invoking $timeout without a delay executes the function directly. Which means you do not need a timeout.